internal/core/algorithm/ngt/option.go

```diff
@@ -123,7 +123,7 @@ func WithDistanceTypeByString(dt string) Option {
 		d = Cosine
 	case "poincare", "poinc", "poi", "po", "pc":
 		d = Poincare
-	case "lorenz", "loren", "lor", "lo", "lz":
+	case "lorentz", "loren", "lor", "lo", "lz":
```

Comment:
We should keep ‘lorenz’ for backwards compatibility.
```suggestion
	case "lorentz", "lorent", "lorenz", "loren", "lor", "lo", "lz":
```